backend-rust/src/indexer.rs

```diff
@@ -526,6 +526,9 @@ impl ProcessEvent for BlockProcessor {
             }
             out.push_str(format!("\n- {}:{}", block.height, block.hash).as_str())
         }
+        process_release_schedules(new_context.last_block_slot_time, &mut tx)
```

Comment:
I assume it has been discussed where this should be placed. 

The alternative of handling this in the query and using an index to query it on seems to be the better solution to me. This would be in particular because if we have an index on release_time querying the value is the same performance as now, the api is less reliant on the indexer to be running with a certain frequency, and we are lowering the extra request per cycle that occurs now. 

If we are afraid of querying up too much data we can just run a periodical job cleaning it up. 

That said this does not seem too unreasonable to me, and I believe since the decision has been made we may want to just live with it.

Reply:
Although the change required to make the change is also rather small

Reply:
Good point with the API now relying on the indexer running for the query to be correct, I'll fix that.

I think cleaning up is still reasonable during processing of the blocks, since the table won't grow unless the indexer is running anyway.